apps/condo/domains/analytics/components/Dashboard.tsx

```diff
@@ -401,7 +401,7 @@ export const Dashboard: React.FC<{ organizationId: string }> = ({ organizationId
             const { result } = response
             setOverview(result.overview)
         },
-        onError: error => {console.log(error)},
+        onError: error => {console.error({ msg: 'Error', error })},
```

Comment:
You need to split error to fields. I think it's 
```suggestion
        onError: e => console.error(JSON.stringify({ msg: 'DashboardError', err: { type: e?.name ?? 'Error', message: e?.message ?? 'Unknown error', stack: e?.stack }) }))

```